use/quickstart.md

```diff
@@ -5,28 +5,171 @@ permalink: /use/quickstart/
 
-These code repositories will be refactored and managed together (along with other code and new contributions) into a single repository.
+1. Download and install [Apache Maven](http://maven.apache.org/download.cgi) by following Maven's [installation guide](http://maven.apache.org/install.html) for your specific operating system.
 
-## Documentation
-* [Apache Beam incubation proposal](https://goo.gl/KJrEl7)
-* *Apache Beam technical vision*
-    * [Detailed](https://goo.gl/5qZt3d)
-    * [Summary](https://goo.gl/nk5OM0)
-* [Apache Beam technical documentation](https://goo.gl/ps8twC)
+
+## Get the WordCount Code
+
+The easiest way to get a copy of the WordCount pipeline is to use the following command to generate a simple maven project that contains Beam's WordCount examples and builds against the most recent Beam release: 
+
+```
+$ mvn archetype:generate \
+      -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-examples \
+      -DarchetypeVersion=LATEST \
+      -DarchetypeGroupId=org.apache.beam \
+      -DgroupId=org.example \
+      -DartifactId=word-count-beam \
+      -Dversion="0.1" \
+      -DinteractiveMode=false \
+      -Dpackage=org.apache.beam.examples
+```
+
+This will create a directory `word-count-beam` that contains a simple pom file and series of example pipelines that count words in text files. 
+
+```
+$ cd beam-word-count/
+
+$ ls
+pom.xml	src
+
+$ ls src/main/java/org/apache/beam/examples/
+DebuggingWordCount.java	WindowedWordCount.java	common
+MinimalWordCount.java	WordCount.java
+```
+
+For a detailed introduction to the Beam concepts used in these examples, see the [WordCount Example Walkthrough](/use/wordcount-example). 
+
+
+## Run WordCount
+
+A single Beam pipeline can run on multiple Beam [runners](/learn/runners), including [Spark](/learn/runners/spark), [Flink](/learn/runners/flink), or [Dataflow](/learn/runners/dataflow). The [DirectRunner](/learn/runners/direct) is a common runner for getting started, as it runs locally on your machine and requires no specific setup.
+
+After you've chosen which runner you'd like to use:
+
+1. Ensure you've done any runner-specific setup.
+1. Build your commandline by:
+   1. Specifying a specific runner with `--runner=<runner>` (defaults to the [DirectRunner](/learn/runners/direct))
+   1. Adding any runner-specific required options 
+   1. Choosing input files and an output location are accessible on the chosen runner. (For example, you can't access a local file if you are running the pipeline on an external cluster.)
+1. Run your first WordCount pipeline.
+
+_([BEAM-902](https://issues.apache.org/jira/browse/BEAM-902): Make these instructions toggle neatly instead of in long lists.)_
+
+1.  Direct Runner
+
+	```
+	$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
+	     -Dexec.args="--inputFile=pom.xml --output=counts"
+	```
+
+1.  Apache Flink Runner
+
+	``` 
+	TODO BEAM-899
```

Comment:
I think it's important to be clear that this guide is not finished and that the lack of those instructions is a (very!) temporary state. Will leave for now and very actively encourage those issues to be fixed ;-)